eosdac-material/src/pages/registercandidate.vue

```diff
@@ -163,10 +168,19 @@ export default {
         }
         requestedpay = requestedpay.toFixed(4)+ ' EOS';
 
+        let stake = this.stakeamount;
+        console.log(stake)
```

Comment:
Might want to comment this out when we're ready or if it's needed for debugging, make it more clear what is being output. Something like `console.log("Stake: ' + stake);`